src/translate.rs

```diff
@@ -61,26 +61,22 @@ pub trait Ptr: Copy {
 }
 
 impl <T> Ptr for *const T {
-    fn is_null(&self) -> bool {
-        (*self).is_null()
-    }
+    #[inline]
+    fn is_null(&self) -> bool { (*self).is_null() }
```

Comment:
Like I said it doesn't matter, I have a personal preference for expanded code but who cares ? ^^